docker/Dockerfile-netty-tcnative-centos6

```diff
@@ -0,0 +1,53 @@
+FROM centos:6
+MAINTAINER netty@googlegroups.com
+
+ENTRYPOINT /bin/bash
+
+ENV SOURCE_DIR /root/source
+ENV MAVEN_VERSION 3.5.0
+ENV JAVA_VERSION 1.8.0
+ENV CMAKE_VERSION 3.8.2
+ENV NINJA_VERSION 1.7.2
+ENV GO_VERSION 1.9.3
+
+# install dependencies
+RUN yum install -y \
+ apr-devel \
+ autoconf \
+ automake \
+ bzip2 \
+ git \
+ glibc-devel \
+ gnupg \
+ java-$JAVA_VERSION-openjdk-devel \
+ libapr1-dev \
+ libtool \
+ lsb-core \
+ make \
+ openssl-devel \
+ perl \
+ tar \
+ unzip \
+ wget
+
+RUN mkdir $SOURCE_DIR
+WORKDIR $SOURCE_DIR
+
+RUN wget -q https://cmake.org/files/v3.8/cmake-$CMAKE_VERSION-Linux-x86_64.tar.gz && tar zxf cmake-$CMAKE_VERSION-Linux-x86_64.tar.gz && mv cmake-$CMAKE_VERSION-Linux-x86_64 /opt/ && echo 'PATH=/opt/cmake-$CMAKE_VERSION-Linux-x86_64/bin:$PATH' >> ~/.bashrc
```

Comment:
`3.8` and `CMAKE_VERSION` have some dependencies ... should there be some state shared here?

Reply:
@Scottmitch good idea!